django/core/validators.py

```diff
@@ -104,7 +107,101 @@ class URLValidator(RegexValidator):
         r'(?:[/?#][^\s]*)?'  # resource path
         r'\Z', re.IGNORECASE)
     message = _('Enter a valid URL.')
-    schemes = ['http', 'https', 'ftp', 'ftps']
+    schemes = [
```

Comment:
I see. So you would like it to reject still valid schemes (such as `file`, `git`), but add the ability to ignore the validation completely?